ext/pgsql/tests/01createdb.phpt

```diff
@@ -9,14 +9,14 @@ pgsql
 // create test table
 
 include('inc/config.inc');
+$table_name = 'table_01createdb';
+$table_name_92 = 'table_01createdb_92';
+$view_name   = "view_01createdb";
 
 $db = pg_connect($conn_str);
 if (!($q = @pg_query($db, "SELECT * FROM ".$table_name)) || !@pg_num_rows($q))
 {
-    pg_query($db,$table_def); // Create table here
-    for ($i=0; $i < $num_test_record; $i++) {
-        pg_query($db,"INSERT INTO ".$table_name." VALUES ($i, 'ABC');");
-    }
+    pg_query($db, "CREATE TABLE {$table_name} (num int, str text, bin bytea)"); // Create table here
```

Comment:
This is not inserting data any more?

Reply:
OK. This is no longer needed.

This test is originally a test for `cratedb` as the test file name suggests.
However, before the fix in this pull request was made, it was also responsible for inserting data used in subsequent tests. I fixed it as follows.

> * Test independence:
> Some tests were dependent on the execution order of previous and subsequent tests.
> All tests can now be executed independently.

In other words, instead of inserting it as a "common process" at this point, it has been modified to insert the data required for each test individually.

Reply:
ACK